src/views/layouts/main.hbs

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" type="text/css" href="/style.css">
-    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="https://unpkg.com/tachyons/css/tachyons.min.css"/>
```

Comment:
Because I was getting a 503 error and it turns out because tachyons version is updated. removed the version for preventing fututre errors like that